lib/recipe.rb

```diff
@@ -34,8 +32,23 @@
 server_uri        = URI(Chef::Config[:chef_server_url])
 trusted_certs_dir = File.directory?(Chef::Config[:trusted_certs_dir]) ? Chef::Config[:trusted_certs_dir] : nil
 
+agent_service_name = value_for_platform_family(
+  %i(windows)            => nil,
+  %i(debian rhel amazon) => 'automate-liveness-agent',
+  %i(freebsd)            => 'automate_liveness_agent'
+)
 init_script_path = value_for_platform_family(
-  %i(debian rhel amazon) => '/etc/init.d/automate-liveness-agent'
+  %i(debian rhel amazon) => "/etc/init.d/#{agent_service_name}",
+  %i(freebsd)            => "/etc/rc.d/#{agent_service_name}"
+)
+install_user = value_for_platform_family(
+  %i(windows)                    => 'administrator',
+  %i(debian rhel amazon freebsd) => 'root'
+)
+install_group = value_for_platform_family(
+  %i(windows)            => 'administrator',
+  %i(debian rhel amazon) => 'root',
+  %i(freebsd)            => 'wheel'
```

Comment:
We talked about just putting the `gid` of 0 in here, not sure if that's better though.